design/Alameda/design.md

```diff
@@ -0,0 +1,30 @@
+## What is Alameda
+
+Alameda is an intelligent resource orchestrator for Kubernetes, providing the features of autonomous balancing, scaling, and scheduling by using machine learning. Alameda learns the continuing changes of computing and I/O metrics from clusters, predicts the future demands for pods, and intelligently orchestrates underlying resources to fulfill the dynamic resource requests without manual configuration.
+
+For more details, please refer to https://github.com/containers-ai/Alameda
+
+## Features
+
+Alameda provides Rook with the following features:
+
+- Disk health prediction  
+    Based on a disk's S.M.A.R.T. value, Alameda predicts how bad a commodity disk is going to fail in a near future. Rook can stop provisioning volumes from a critical status disk.
+- Performance prediction  
+    Alameda learns patterns from the historical performance metrics of persistent volumes and pools. With this knowledge, Rook can:
```

Comment:
would be great if expanding what metrics can be learned and how they are collected

Reply:
Please see the refreshed design.md.